tests/test_changeset.py

```diff
@@ -436,8 +436,8 @@ def test_faulty_simple_attributes_log_AttributeError(
         titem = tracker["items"][0]
         titem["imagination"] = 1
         message_end = (
-            "Invalid module 'project/space/example title'. Invalid "
-            "workitem 'REQ-001'. imagination isn't defined on Folder"
+            "Invalid module 'project/space/example title'. Invalid workitem "
+            "'REQ-001'. 'Folder' object has no attribute 'imagination'"
```

Comment:
We probably shouldn't assert on the exact text of someone else's error to begin with — rather, we should only check that it contains some relevant keywords (in this example, "REQ-001", "Folder" and "imagination").